sdk/02_tutorial/07_verification.md

```diff
@@ -11,59 +11,71 @@ import TabItem from '@theme/TabItem';
+
 In this section, you play the role of a <span className="label-role verifier">Verifier</span> that does the following:
 
-1. Take a `Presentation` object supplied by a <span className="label-role claimer">Claimer</span>
+:::info
+
+The Verifier also needs an account and DID, but the tutorial omits these steps.
+
+:::
+
+1. Take a `VerifiablePresentation` object supplied by a <span className="label-role holder">Holder</span>
 2. Verify that its data is correct
-3. Verify that the attestation is valid, i.e., its hash exists on-chain and the attestation has not been revoked
-4. Verify that the <span className="label-role claimer">Claimer</span> sending the `Credential` owns it
+3. Verify that the presentation is valid, i.e., its hash exists on-chain and the presentation hasn't been revoked
+4. Verify that the <span className="label-role holder">Holder</span> sending the `Credential` owns it
 
-:::info Presentation object
+:::info VerifiablePresentation object
 
-The <span className="label-role claimer">Claimer</span> uses a Credential to create the `Presentation` object.
-Unlike the credential, a `Presentation` can hide some attributes that are not required by the <span className="label-role verifier">Verifier</span> and can contain a claimer-signed challenge.
-A `Presentation` also contains a proof that the <span className="label-role claimer">Claimer</span> owns the credential.
+The <span className="label-role holder">Holder</span> uses a Credential to create the `VerifiablePresentation` object.
+Unlike the credential, a `VerifiablePresentation` can hide some attributes that aren't required by the <span className="label-role verifier">Verifier</span> and can contain a holder-signed challenge.
+A `VerifiablePresentation` also contains a proof that the <span className="label-role holder">Holder</span> owns the credential.
 
 :::
 
-## Create Presentation
+## Verify presentation
 
-A <span className="label-role claimer">Claimer</span> needs to send more than a credential, as they also need to prove ownership of the credential.
-A <span className="label-role claimer">Claimer</span> does this by creating a presentation and signing the <span className="label-role verifier">Verifier</span>'s challenge.
 
-<TsJsBlock fileName="claimer/createPresentation">
-  {CreatePresentation}
-</TsJsBlock>
 
-The `createPresentation` method returns a presentation, taking the credential, a callback to sign data, and the <span className="label-role verifier">Verifier</span>'s challenge as input.
+The KILT SDK provides the `verifyPresentation` method from the `Verifier` class to verify a credential presentation. It takes the following parameters:
+
+- `presentation`: The `VerifiableCredential` to verify.
+- `verficationCriteria`: An object of criteria that need to pass to verify the presentation. These can be any combination of criteria.
+<!-- TODO: Find out more link -->
 
-## Verify
+The method returns a boolean value indicating whether the presentation is valid.
```

Comment:
it's an object, and one of this object's properties (`verified`) has a boolean value indicating whether the presentation and credential are valid